### ISSUE TITLE
Check namespace aliases for single level renames?

### ISSUE THREAD

**Issue by padraic** [humbug/php-scoper#39]
While pointless single level namespaces will raise a warning in use, there is an exception for setting an alias. As equally pointless as that may be...
```
<?php
use Symfony; // raise warning
use Symfony as Laravel; // no warning
new Laravel\Component\Finder\Finder; // Symfony Finder instance
```
With the most recent change in scoping, any `use XXX as YYY` would escape scoping for now.

**Comment by theofidry** [comment]
@padraic I would generate a unique alias then, but 👍

**Comment by padraic** [comment]
Closed via #50
